ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -71,7 +71,7 @@ trait CommonFtpStageSpec extends BaseSpec with Eventually {
   implicit val system = getSystem
   implicit val mat = getMaterializer
   implicit val defaultPatience =
-    PatienceConfig(timeout = Span(10, Seconds), interval = Span(300, Millis))
+    PatienceConfig(timeout = Span(30, Seconds), interval = Span(600, Millis))
```

Comment:
Actually I've seen tests be quite slow on my local machine at times as well - but haven't taken the time to really dig into that yet.